pygameweb/templates/error.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>{{ code }} - pygame.org</title>
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+
+    <style>
+        * {
+            font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+            text-align: center;
+            color: #111;
+        }
+
+        h1 {
+            margin-top: 128px;
+            font-size: 10em;
+            margin-bottom: 16px;
+        }
+
+        h2 {
+            font-size: 1.9em;
+            font-weight: 400;
+
+            padding: 0 64px;
+        }
+
+        h3 {
+            font-weight: 400;
+        }
+
+        p {
+            font-size: 1.2em;
+            color: #444;
+        }
+
+        a {
+            color: #337ab7;
+            text-decoration: none;
+        }
+
+        img {
+            height: 100px;
+        }
+    </style>
+
+</head>
+<body{% block body_attribs %}{% endblock body_attribs %}>
+
+<div id="container">
+    <img src="http://www.pygame.org/docs/pygame_logo.gif">
```

Comment:
We should use https here.

Reply:
When I loaded the image initially the site didn't redirect off http so I forgot to change it. I'll swap it out, but that's also an issue that needs addressed as `http://...` should be forcefully redirecting to `https://...`

Reply:
We have to leave http on for now... for these reasons: http://renesd.blogspot.com/2018/02/why-cant-we-turn-off-http.html